docs/source-app/api_reference/api/lightning_app.core.LightningApp.rst

```diff
@@ -1,10 +0,0 @@
-﻿.. role:: hidden
```

Comment:
can't find this, where was it moved?

Reply:
This was removed as it duplicated the generated on and caused an issue while building docs, in fact, that shall be removed a while back as this was a bad artifact...